04_numeronym/srini/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var testMatrix = []struct {
+	input  []string
+	output []string
+}{
+	{[]string{"abba", "accessibility", "Kubernetes", "abc"}, []string{"a2a", "a11y", "K8s", "abc"}},
+	{[]string{"abb a"}, []string{"a3a"}},
+	{[]string{""}, []string{""}},
+	{[]string{"£€€€§‡®"}, []string{"£5®"}},
+	{[]string{"ßäöüÄÖÜ"}, []string{"ß5Ü"}}}
```

Comment:
Please also add some tests with whitespace at the beginning/end/middle of string. Sorry for not thinking of this earlier.

Reply:
No worries, added :)

Reply:
Bonus points for using German characters 😼